zson/parser-types.go

```diff
@@ -0,0 +1,367 @@
+package zson
```

Comment:
parsertypes.go would be a more conventional name.

Reply:
See issue #1825